libs/ledger-live-common/src/families/stellar/api/horizon.ts

```diff
@@ -66,7 +77,7 @@ export const fetchBaseFee = async (): Promise<{
     };
   }
 
-  const baseFee = StellarSdk.BASE_FEE || FALLBACK_BASE_FEE;
+  const baseFee = Number(BASE_FEE) || FALLBACK_BASE_FEE;
```

Comment:
While `Number` may work, I advise just using `new BigNumber()` everywhere. It's the pattern LL uses for now, until we migrate away from using `BigNumber` and probably use `BigInt` instead.

Using Number's constructor is even discouraged: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/Number#return_value

Reply:
Yeah I instead tried to follow what was already used in the file but I can use Bignumber (even though it might be overkill and parseInt should be enough)